.travis.yml

```diff
@@ -70,6 +70,20 @@ jobs:
     deploy:
     - provider: script
       script: .travis_scripts/deploy_docker.sh dev
+  - stage: Deploy Swagger file
+    if: tag IS present
+    deploy:
+    - provider: s3
+      access_key_id: AKIAI3FDTJG3O6WWN4BQ
+      secret_access_key:
+        secure: "mJtZqcGrs8bTJKEaV/VkUfcVTT4y+U1s6hztF45diTDQpT0uleLXClO8ljj3S0mbN3r+ykLBRvSohOOki94+v8z9/VDptVyrFUNEah2PKl30D3S74LeRXN/3xK25h6Y9HLdcbTuhlY988sQGONztkpS7kQCnaDa38rB0eUXfqhBC6WRJpmVmp1T0OSReZM7Et8VFfzcjudxQDT3YuYAvSd3MNpgfq6YXRdetzB0Tou8PtdyO02C87JANw9+BYxgaEcNtVIAtkEHyFriOdkqeo9c0zBJJbU/B6FxC/XZL6PfakZnVTuNEHjDuOvPhjMHKg1yWuCLhWchY9aZ3tbm9A7QG0cVycb/dwGeyNCwwocjPRMKell+2a749sYvP5WqNjYD+0wZp54WCglzTOxXy2EzX4UJZ6EeDzMOEJJ/Z/BnTZaRk9PiaH8iCSLOPqZ9uQV9ZV4+HliQYg/6yV33gT1PjjV6B80omYdrzu5kL2pZZa8w7cJ0jrbf7Li/Abhs8E6c7Wi11wtJUmzLPEl/uFl2XoKIMC4VWwymwB20JIlvD7QY+BnMB9WkTgYHYtDzwaZfFZqWybnOth2zRstA5kpakpEgZ34IslIDLRrXdjJeZtSiMz7/sgq5RNK7L3xzXTu4D+DiDzylNnOq1urL3qaQeHzMyME2OGvuxit1okR8="
+      bucket: eu-west-1-streamr-cdn
+      acl: private
+      region: eu-west-1
+      skip_cleanup: true
```

Comment:
This should be `cleanup: true` according to Travis build config validation (dpl v2) https://travis-ci.com/github/streamr-dev/engine-and-editor/builds/162403546/config

Reply:
dplv2 is still in beta.